mods/ts/mod.yaml

```diff
@@ -148,7 +148,7 @@ ChromeLayout:
 	ts|chrome/mainmenu-prerelease-notification.yaml
 	common|chrome/settings.yaml
 	common|chrome/credits.yaml
-	common|chrome/lobby.yaml
+	ts|chrome/lobby.yaml
```

Comment:
Why does this need to be cloned?

Reply:
I need to set a property indivudually on the SERVER_NAME Widget of the lobby.yaml. This property is unique to the TS mod.

I couldn't find a way to do this without cloning the whole file, is there an easier way?

Reply:
This highlights a limitation of the chrome metrics - they set a single value for a property but often times we may need more than one. Just like here where a different highlight color is needed depending on the background color. It is also a limitation of chrome yaml - some kind of yaml inheritance for defining components or default styles/properties would be useful here.

Reply:
I was thinking about this and it feel a little like what CSS was invented to solve in HTML, applying properties selectively to elements in a tree without having to copy the whole (or parts of) the tree.